bitnami/external-dns/templates/NOTES.txt

```diff
@@ -0,0 +1,45 @@
+{{- if eq .Values.provider "aws" }}
+{{- if or (empty .Values.aws.accessKey) (empty .Values.aws.secretKey) }}
+  ###############################################################################
+  ### ERROR: You chose 'aws' as your DNS provider and you did not set the     ###
+  ### AWS access&secret keys                                                  ###
+  ###############################################################################
+
+Indicate your AWS access&secret keys by installing this chart as shown below:
+
+    $ helm install --name my-release bitnami/external-dns --set provider='aws',aws.accessKey='YOUR_ACCESS_KEY',aws.secretKey='YOUR_SECRET_KEY'
```

Comment:
I think that telling the user to upgrade the chart would be better than deploying a completely new one. That's what we do in other charts AFAIK

Reply:
👍